apiserver/facades/agent/uniter/uniter.go

```diff
@@ -1046,6 +1046,38 @@ func (u *UniterAPI) JoinedRelations(args params.Entities) (params.StringsResults
 	return result, nil
 }
 
+// Refresh updates data saved by the uniter units.
```

Comment:
// Refresh retrieves the latest values for attributes on this unit

(or something like that)

Reply:
done